src/utils.rs

```diff
@@ -47,6 +48,18 @@ pub fn get_source(input: &str) -> Source {
     }
 }
 
+pub fn render_one_off_template(
+    content: &str,
+    context: &Context,
+    path: Option<PathBuf>,
+) -> Result<String> {
+    let mut tera = Tera::default();
```

Comment:
there's actually a better way to do that now: https://docs.rs/tera/latest/tera/struct.Tera.html#method.one_off

Reply:
question: should we set the `autoscape` to `true` or transform it into a function parameter?

Reply:
no autoescape needed there

Reply:
@Keats autoscape set to `false`!